README.md

```diff
@@ -22,50 +37,78 @@ This is my personal config in windows11 and ubuntu.
 11. f.lux
```

Comment:
I remember this software seems to be open-source. If it's not open-source, then add the link to the official website. If it is an open-source project, add the GitHub link, just like the code in the other lines.